common/lib/src/i18n/string.dart

```diff
@@ -1,4 +1,10 @@
 import 'package:intl/intl.dart';
+import 'package:flashcards_common/src/i18n/generated/messages_all.dart';
+
+void initLocales(List<String> locales) {
```

Comment:
This should not be part of strings. We should move it to another file (don't forget to export it in common/i18n.dart) :-)